src/main/kotlin/uk/gov/justice/hmpps/offenderevents/services/HMPPSDomainEventsEmitter.kt

```diff
@@ -164,23 +162,24 @@ class HMPPSDomainEventsEmitter(
       .withAdditionalInformation("currentPrisonStatus", receivedReason.currentPrisonStatus?.name)
   }
 
-  private fun toMergedOffenderNumbers(event: PrisonerMergedOffenderEvent): List<HmppsDomainEvent> {
-    val mergeResults = mergeRecordDiscriminator.identifyMergedPrisoner(event.bookingId)
```

Comment:
main change; no need to look up if there is a previous merge identifier since the event now contains information about the merge.

This fixes issue where the event was incorrectly being raised after a book number change that happened sometime after a merge - and we would raise 2 merge events in this scenario